force-app/main/default/objects/Medication/fields/Initial_Recommended_Dosage__c.field-meta.xml

```diff
@@ -4,9 +4,8 @@
     <encryptionScheme>None</encryptionScheme>
     <externalId>false</externalId>
     <label>Initial Recommended Dosage</label>
-    <length>255</length>
-    <required>false</required>
+    <length>131072</length>
```

Comment:
This is 40 pages of type, the expectation is that fields are appropriately sized, is this approved as necessary by the client? @Sumancdey @ckleinacn